pyproject.toml

```diff
@@ -141,22 +140,24 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-aarch64-image = "manylinux2014"
 before-build = "bash {project}/tools/wheels/cibw_before_build_linux.sh {project}"
 
+[tool.cibuildwheel.linux.environment]
+# /project will be the $PWD equivalent inside the docker used to build the wheel
+PKG_CONFIG_PATH="/project/"
+
 [tool.cibuildwheel.macos]
-before-build = "bash {project}/tools/wheels/cibw_before_build_macos.sh {project}"
+before-build = "pip install ninja; bash {project}/tools/wheels/cibw_before_build_macos.sh {project}"
+
+[tool.cibuildwheel.macos.environment]
+# On macos-arm64, use Accelerate instead of setting this
+# CIBW_CONFIG_SETTINGS=\"setup-args=-Dblas=accelerate
+PKG_CONFIG_PATH="{project}"
 
 [tool.cibuildwheel.windows]
-before-build = "bash {project}/tools/wheels/cibw_before_build_win.sh {project}"
+before-build = "pip install ninja && bash {project}/tools/wheels/cibw_before_build_win.sh {project}"
```

Comment:
This change should not be needed. Probably was a temporary change when you had build isolation disabled?

Reply:
Let's see what happens if I remove it. Does anything else install ninja on windows and macos?

Reply:
Yes, `meson-python` will unless build isolation is disabled. CI seems happy, so resolving.